Automattic/studio
# Studio: Block the text input when sending

Fixes https://github.com/Automattic/dotcom-forge/issues/7932

## Proposed Changes

When the user sends a message, the input is not disabled. This PR fixes that.

## Testing Instructions



1. Naviate to assistant tabe
2. Send a message
3. Ensure that while the thinking bubble appears the input is disabled

## Pre-merge Checklist
